pkg/services/stats/statsimpl/stats.go

```diff
@@ -144,6 +143,21 @@ func (ss *sqlStatsService) GetSystemStats(ctx context.Context, query *stats.GetS
 		return nil
 	})
 
+	// Execute dashboard size analysis in its own query (these may take time)
+	if err == nil {
+		var rows *sql.Rows
+		rows, err = ss.db.GetSqlxSession().Query(ctx, `SELECT `+
+			`(SELECT SUM(LENGTH(data)) FROM `+dialect.Quote("dashboard")+` WHERE is_folder = ?) AS dashboard_bytes_total,`+
+			`(SELECT MAX(LENGTH(data)) FROM `+dialect.Quote("dashboard")+` WHERE is_folder = ?) AS dashboard_bytes_max`,
+			dialect.BooleanStr(false), dialect.BooleanStr(false))
```

Comment:
We can make this more efficient if we gather both stats in a single select so we only have to scan the table once. The original arrangement used multiple subqueries because each one could potentially be querying a different table.
```suggestion
		rows, err = ss.db.GetSqlxSession().Query(ctx, `SELECT SUM(LENGTH(data)) AS dashboard_bytes_sum, MAX(LENGTH(data)) AS dashboard_bytes_max FROM `+dialect.Quote("dashboard")+` WHERE is_folder = ?`,
			dialect.BooleanStr(false))
```